Automattic/wp-calypso
# Signup: template-first theme Signup flow

This adds a Signup flow that allows a user to select a template-first theme instead of having one auto-assigned via segment and topic during Signup.

Epic: https://github.com/Automattic/zelda-private/issues/119

<img width="1407" alt="Screen Shot 2019-10-21 at 2 15 29 PM" src="https://user-images.githubusercontent.com/942359/67231478-5f2c4600-f40d-11e9-9889-6675432a4954.png">

**To Test:**
- navigate to `/start/design-first/`
- you should see a screen with 18 template-first themes
- select a theme
- finish Signup and verify that the correct theme is set with content similar to the screenshot
- navigate to `/start/domain/`
- create a new site, when you get to the themes step, you should only see 3 themes (based on the selection from the previous step)
- finish Signup and verify that the correct theme is set
- navigate to `/start/domain` and create a domain only site
- click the **Create Site** button on the domain management screen
- go through the `site-selected` flow, picking one of the 3 themes
- Finish flow. Site should be created, but it's not headstarted with the content from the theme (may or may not be a bug, but it's what happens on production)
